fastlane_bot/events/interface.py

```diff
@@ -216,6 +216,7 @@ def remove_zero_liquidity_pools(self) -> None:
                 exchanges.append(ex)
                 keys.append(["liquidity"])
             elif ex in self.cfg.CARBON_V1_FORKS:
+                print(f"[events.interface] Removing zero liquidity pools for {ex}")
```

Comment:
Just a reminder to remove print statement

Reply:
removed, thx